crispy_tailwind/templates/tailwind/layout/select.html

```diff
@@ -2,12 +2,12 @@
 {% load l10n %}
 
 <div class="relative">
-<select class="{% if field.errors %}border border-red-500 {% endif %}bg-white focus:outline-none border border-gray-300 rounded-lg py-2 px-4 block w-full appearance-none leading-normal text-gray-700" name="{{ field.html_name }}" {{ field.field.widget.attrs|flatatt }}>
+<select id= "{{ field.name }} class="{% if field.errors %}border border-red-500 {% endif %}bg-white focus:outline-none border border-gray-300 rounded-lg py-2 px-4 block w-full appearance-none leading-normal text-gray-700" name="{{ field.html_name }}" {{ field.field.widget.attrs|flatatt }}>
```

Comment:
this should read:
```suggestion
<select id="{{ field.name }}" class="{% if field.errors %}border border-red-500 {% endif %}bg-white focus:outline-none border border-gray-300 rounded-lg py-2 px-4 block w-full appearance-none leading-normal text-gray-700" name="{{ field.html_name }}" {{ field.field.widget.attrs|flatatt }}>
```

but even with the format fixed this would only be valid if the field name is unique for the whole page, and if it follows the rules for HTML IDs

Reply:
using `field.html_name` would probably be correct (it’s used for the `name` attribute, which serves a different role than the `id` attribute)